.circleci/test-api.sh

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env bash
+
+apt-get update
+apt-get install -y jq
+
+make serve-local &
+
+curl --retry-connrefused \
+    --retry 5 \
+    --retry-delay 20 \
```

Comment:
20 seconds of delay seems long, especially if we try 5 times: if the result is negative, we will systematically have a delay of 20×5 = 100 seconds = 1 minute 40 seconds.

Choose instead the median delay for the API start-up  :)
```suggestion
    --retry-delay 5 \
```

Reply:
I updated it to --retry 10, because 5 is not enough for the server to start up.